fhir-model/src/main/java/com/ibm/fhir/model/lang/util/LanguageRegistryUtil.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.model.lang.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Utility class for language registry.
+ */
+public class LanguageRegistryUtil {
```

Comment:
have you thought about calling this in the FHIRServletContextListener? that way it's activated on startup

Reply:
Didn't really know about it.  But I can.

Reply:
give it a peek - if it makes sense to activate and load before any requests, that's where to put it

Reply:
Added.